lanelet2_python/python_api/io.cpp

```diff
@@ -51,13 +79,13 @@ BOOST_PYTHON_MODULE(PYTHON_API_MODULE_NAME) {  // NOLINT
       "Loads a map robustly. Parser errors are returned as second member of "
       "the tuple. If there are errors, the map will be incomplete somewhere.");
 
-  def("write", writeProjectorWrapper, (arg("filename"), arg("map"), arg("projector") = DefaultProjector()),
+  def("write", writeProjectorWrapper, (arg("filename"), arg("map"), arg("projector"), arg("params") = Optional<io::Configuration>{}),
```

Comment:
This default has never actually worked. Try is on master and you'll get an error. So, I removed it because I couldn't get the conversions for it working.

Reply:
Valid point 😄. Lets keep it like this then. I was never happy with this default anyways as we are discuraging using the spherical mercator in all places except this one.